### PR TITLE
Add filter for All CMSs in getting earnings

### PR DESCRIPTION
**Pull request by sibayanjasper**
### a. Implementation Changes
- Add All CMSs to filtering by Asset CMS in Earnings page

### a. Ticket link/Request Feature link
- [Dev-826](https://freedom.myjetbrains.com/youtrack/issue/Dev-826)

### CODE REVIEW

File: app/scripts/components/earnings/index.js [hunk 1]
```diff
@@ -239,7 +239,9 @@ var React = require('react'),
 
         render: function () {
             var months = '',
-                asset_cms,
+                asset_cms = {
+                    all: 'All CMSs'
```
Comment (wilbertverayin) [inline app/scripts/components/earnings/index.js]: why not put to translation?

File: app/scripts/components/earnings/index.js [hunk 2]
```diff
@@ -269,7 +271,9 @@ var React = require('react'),
                 </div>
             );
 
-            asset_cms = _.map(this.state.asset_cms, function (value, key) {
+            asset_cms = _.assign(asset_cms, this.state.asset_cms);
+
+            asset_cms = _.map(asset_cms, function (value, key) {
```
Comment (wildtron) [inline app/scripts/components/earnings/index.js]: isn't this chainable?

Comment (sibayanjasper) [inline app/scripts/components/earnings/index.js]: _updated_

File: app/scripts/components/earnings/index.js (hunk 1 above)
Comment (sibayanjasper) [inline app/scripts/components/earnings/index.js]: _updated_

File: app/scripts/components/earnings/video_cut.js [hunk 3]
```diff
@@ -16,13 +16,13 @@ var React = require('react'),
...
         render: function () {
-            var data = _.round(this.props.data, 2);
+            var data = _.round(this.props.data, 4);
 
             if (!this.props.rowData.user_id && data) {
                 return (
                     <a className="griddle-link"
                        onClick={this.handleGetVideoCutBreakdown}>
-                        {data}
+                        {data.toFixed(4)}
```
Comment (cedricabuso) [inline app/scripts/components/earnings/video_cut.js]: Are we sure that all this variables with `toFixed` are always numbers and not strings?

Comment (sibayanjasper) [inline app/scripts/components/earnings/video_cut.js]: Yes. Lodash round returns numbers.

**Pull request merged**